radixdlt/src/main/java/com/radixdlt/consensus/validators/Validator.java

```diff
@@ -85,46 +64,35 @@
 	private final boolean jailed;
 */
 
-    // Public key for consensus
-	private ECPublicKey nodeKey;
+	// Power associated with each validator, could e.g. be based on staked tokens
+	private final UInt128 power;
```

Comment:
As discussed, I think this is going to be very much easier in future if we use a UInt256 here.